test/IntegrationTests/MSTest.Acceptance.IntegrationTests/GenericTestMethodTests.cs

```diff
@@ -20,57 +20,61 @@ public async Task TestDifferentGenericMethodTestCases()
         testHostResult.AssertExitCodeIs(ExitCodes.AtLeastOneTestFailed);
         testHostResult.AssertOutputMatchesRegex(
             """
-            failed AMethodWithBadConstraints \(0\) \((\d+s )?\d+ms\)
+            failed AMethodWithBadConstraints \(0\) \((?:\d+s\s?)?(?:\d+ms)\)
```

Comment:
Isn't the previous regex more readable? The part of an optional "seconds" component of duration is already fixed. So I guess the changes here are unnecessary.

Reply:
Fine by me, reverting this part. Note that it was done before you fixed something (PR is open for a bit now)